qa/standalone/scrub/osd-scrub-dump.sh

```diff
@@ -138,7 +138,7 @@ function TEST_recover_unexpected() {
 	    break
 	fi
 	total=$(expr $total + $pass)
-	sleep $(expr $SCRUB_SLEEP \* 2)
+	sleep $(expr $SCRUB_SLEEP \* 2 - 2)
```

Comment:
?

Reply:
I've accidentally removed the relevant part in the commit message. The sleeps, esp. the first one, were too long
(as evident from the gist linked in the PR message).
I'll restore the explanation.

Reply:
I actually kept the full explanation in the PR description.

Reply:
... and simplify the code